games/s.yaml

```diff
@@ -1045,6 +1045,23 @@
   url: http://www.proasm.com/sw/swp.html
   video:
     youtube: JePqKJd4hhw
+    
+- name: SwtPacMan
+  type: clone
+  originals:
+  - Pac-Man
+  repo: https://github.com/fogleman/SwtPacMan
+  url: https://www.michaelfogleman.com/projects/pac-man/
+  development: halted
+  status: playable
+  lang: Java
+  framework: SWT
+  license:
+  - MIT
```

Comment:
I can't find this license in the project pages; where is it?

Reply:
My bad! Michael Fogleman's other projects (Craft, Future Blocks, etc.) released with MIT license. I don't checked this project. I change it to As-is